readme.md

```diff
@@ -2,15 +2,16 @@
 
 1. create database **'edge-node-auth-service'**
 2. cp .env.example .env
-3. generate random strings for following .env variables: **JWT_SECRET** and **SECRET** (you can use openssl rand -hex 64 for example)
+3. generate random strings for following .env variables: **JWT_SECRET** and **SECRET** (you can use openssl rand -hex 64 for example) + Add MySQL password to **DB_PASSWORD** variable if necessary
```

Comment:
i think you can create a new step before this one, saying something like "update your db user config" or something like that